### PR TITLE
prtable: Add second table for open PRs.

### PR DESCRIPTION
**Pull request by deekoder**

### CODE REVIEW

File: getPRs.go [hunk 1]
```diff
@@ -0,0 +1,428 @@
+package main
```
Comment (harshavardhana) [inline getPRs.go:17]: Missing license header..

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: getPRs.go (hunk 1 above)
Comment (deekoder) [reply getPRs.go:17]: oh yes.

Comment (deekoder) [reply getPRs.go:17]: done.

Comment (harshavardhana) [reply getPRs.go:17]: actually this is not the header.. you can take an example from minio server project.

Comment (harshavardhana) [reply getPRs.go:17]: This is BSD-3 clause header.

Comment (deekoder) [reply getPRs.go:17]: My bad. Done.

File: getPRs.go [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Below main package has canonical imports for 'go get' and 'go build'
```
Comment (harshavardhana) [inline getPRs.go]: You don't need this.. this is specific to minio server only.

File: getPRs.go [hunk 3]
```diff
@@ -0,0 +1,450 @@
+/*
+ * Minio Cloud Storage, (C) 2016,2017 Minio, Inc.
```
Comment (harshavardhana) [inline getPRs.go]: Keep it `Buggy, (C) 2017 Minio, Inc.`

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: getPRs.go (hunk 3 above)
Comment (deekoder) [reply getPRs.go]: Done

File: getPRs.go (hunk 2 above)
Comment (deekoder) [reply getPRs.go]: ugh missed this. Done.

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**